src/app/page.tsx

```diff
@@ -2,27 +2,47 @@ import Image from "next/image";
 
 export default async function Home() {
   const res = await fetch(
-    "https://www.yr.no/api/v0/locations/2-2666772/forecast"
+    "https://www.yr.no/api/v0/locations/2-2666772/forecast",
+    {
+      cache: "no-store",
+    }
   );
   const weatherData = await res.json();
 
+  const resCurrent = await fetch(
+    "https://www.yr.no/api/v0/locations/2-2666772/forecast/currenthour",
+    { cache: "no-store" }
+  );
+  const currentWeatherData = await resCurrent.json();
+
   function getDayName(daysInFuture: number) {
-    const daysOfWeek = [
-      "Sunday",
-      "Monday",
-      "Tuesday",
-      "Wednesday",
-      "Thursday",
-      "Friday",
-      "Saturday",
-    ];
+    const daysOfWeek = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
 
     const futureDate = new Date();
     futureDate.setDate(futureDate.getDate() + daysInFuture);
 
     return daysOfWeek[futureDate.getDay()];
   }
 
+  function roundNumber(num: number) {
+    return Math.round(num);
+  }
+
+  function getCurrentHour() {
+    const now = new Date();
+    let currentHour = now.getHours();
+    let currentMinute = now.getMinutes();
+
+    return `${currentHour < 10 ? "0" : ""}${currentHour}:${
+      currentMinute < 10 ? "0" : ""
+    }${currentMinute}`;
+  }
+
+  function getHourFromData(data: any) {
+    const date = new Date(data);
```

Comment:
Think you misspelled this one? It says "data" instead of "date"? Would also change the type to `string` (if that's what it expects) instead of `any`.